RELEASE_NOTES.rst

```diff
@@ -6,14 +67,14 @@ Bugfixes:
 - Remark dependency has been pinned back to ~5.4 because version 6 changes
   current behavior and breaks tests. This is a hotfix which will be rolled
   back when the upstream addresses this.
- 
+
 coala-bears 0.10.0
 ==================
 
 This release, we had 97 commits contributed by 42 contributors.
 
 The coala website has been updated to a new, more modern design. You can now
-find a quick overview of bears and their settings at 
+find a quick overview of bears and their settings at
```

Comment:
If this is being merged to master, a proper fix is to prevent it occurring again by setting up a bear.